src/main/java/org/springframework/data/tarantool/core/query/TarantoolQueryMethod.java

```diff
@@ -26,20 +26,6 @@ public TarantoolQueryMethod(Method method, RepositoryMetadata metadata, Projecti
         this.metadata = metadata;
     }
 
-    public Optional<String> getSpaceName() {
```

Comment:
And where did it go? So we won't have an option of specifying the space name in the `@Tuple` annotation?

Reply:
In the `@Query` method, we can't, since we removed the logic with `@Tuple`. In Entity we can get the name of the space